presto-docs/src/main/sphinx/cache/local.rst

```diff
@@ -0,0 +1,69 @@
+=====================
+Alluxio Cache Library
+=====================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
+Presto supports caching input data locally with a built-in cache
+to reduce query latency using Hive Connector.
+This cache, based on Alluixo client-side cache library,
+utilizes the local storage (such as SSD) with configurable capacity and locations.
+To understand its internals and the benchmark results of latency improvement,
+please read this `article <https://prestodb.io/blog/2020/06/16/alluxio-datacaching>`_.
+Note that this is a **read-cache**, **completely transparent** from users
+and **fully managed** by individual Presto workers.
+To provide Presto an independent, distributed cache service for read/write workloads with
```

Comment:
I am not sure if it is the correct place to talk about the Alluxio cache as a service. WDYT @highker